src/containers/Application.js

```diff
@@ -570,6 +570,15 @@ class Navigation {
 	removeItem(item) {
 		this.Items = this.Items.filter(i => i.name != item.name);
 	}
+
+	removeChildren(children) {
```

Comment:
I don't think that this is a correct way how to implement this.

Technically, you don't remove items/children but you hide them; or it could be implemented as hidden implicitly and shown when resource is granted.